sydent/sms/openmarket.py

```diff
@@ -14,9 +14,8 @@
 
 import logging
 from base64 import b64encode
-from typing import TYPE_CHECKING, Dict, Generator, Optional
+from typing import TYPE_CHECKING, Dict, NoReturn, Optional
```

Comment:
Since we're not using this anymore.

```suggestion
from typing import TYPE_CHECKING, Dict, Optional
```

Reply:
Ha! Flake8 just caught that, I pushed another fix like 2 mins ago resolving that.